src/states/node.rs

```diff
@@ -490,36 +443,6 @@ impl Node {
         self.disconnect_peer(&pub_id)
     }
 
-    fn handle_connect_success(&mut self, pub_id: PublicId, outbox: &mut EventBox) {
-        if self
-            .peer_mgr
-            .get_peer(&pub_id)
-            .map_or(false, Peer::is_routing)
-        {
-            warn!(
-                "{} Received ConnectSuccess from {:?}, but node is already in routing \
-                 state in peer_map.",
-                self, pub_id
-            );
-            return;
-        }
-
-        self.peer_mgr.connected_to(&pub_id);
-        debug!("{} Received ConnectSuccess from {}.", self, pub_id);
-        self.process_connection(pub_id, outbox);
-    }
-
-    fn handle_connect_failure(&mut self, pub_id: PublicId, outbox: &mut EventBox) {
-        if let Some(&PeerState::CrustConnecting) = self.peer_mgr.get_peer(&pub_id).map(Peer::state)
-        {
-            debug!("{} Failed to connect to peer {:?}.", self, pub_id);
-        }
-        let _ = self.dropped_peer(&pub_id, outbox, true);
-        if self.chain.is_member() && self.chain.our_info().members().contains(&pub_id) {
```

Comment:
Lines 518-520 didn't make it to proving_node.rs handle_connect_failure.
I would like to confirm it's intentional.

Reply:
`ProvingNode` doesn't have neither `Parsec` nor `Chain`, so those lines make no sense for it.

Reply:
Another way to look at it is that anything that is behind `if self.chain.is_member() { ... }` doesn't belong to `ProvingNode`, because node in that state isn't a full section member yet.

Reply:
Looks intentional: we can't vote Offline for someone who's still `ResourceProving`. Also, I see this remains the same in this state (way later in this file)

Reply:
Yes, sorry: I realised this later (Had posted prematurely by hitting the wrong button :p)